instance/models/instance.py

```diff
@@ -473,6 +468,18 @@ class AnsibleInstanceMixin(models.Model):
     class Meta:
         abstract = True
 
+    def save(self, **kwargs):
```

Comment:
@smarnach That's much better than calling the pre_save hook - silly me for not thinking of this!

Thanks for this change, and for adding the defaults here.